pucas/tests.py

```diff
@@ -17,6 +17,29 @@
 from pucas.management.commands import ldapsearch, createcasuser
 
 
+class MockLDAPInfo(object):
+    '''Simulate ldap result object with ldap3 specific behavior for getattr'''
+    def __init__(self, **kwargs):
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+    def __getattr__(self, attr):
+        if attr in self.__dict__:
+            return self.__dict__[attr]
```

Comment:
Codecov thinks this line isn't being run (which seems suspicious)

Maybe this is related to your note about the Mock return?